docs/lib/analytics/fragments/autotrack.md

```diff
@@ -3,9 +3,8 @@ The Amplify analytics plugin records when an application opens and closes. This
 To view this from the Console:
 
 ```bash
-amplify console
+amplify console analytics
 ```
 
-1. Navigate to the *Analytics* tab, and then choose *View in Pinpoint*.
-2. On the Pinpoint Console under Analytics, choose *Events*. 
-3. Enable filters, you can select `Session Start` and `Session Stop` events to filter on session events.
+1. On the Pinpoint Console under Analytics, choose *Events*. 
```

Comment:
If the user needs to do something, make it bold so it stands out, so: **Events**

Reply:
Made it bold